### PR TITLE
Hide the KML/GPX download feature when offline

### PR DESCRIPTION
**Pull request by gberaudo**
See https://jira.camptocamp.com/browse/GSLUX-109.

### CODE REVIEW

File: geoportailv3/static/less/offline.less [hunk 1]
```diff
@@ -104,6 +104,9 @@ body.offline-or-disconnected {
   .feature-popup-footer .dropup {
     display: none;
   }
+  .feature-popup-footer .dropdown {
+    display: none;
+  }
```
[the remark this answers is not in the document]
Comment (gberaudo) [reply geoportailv3/static/less/offline.less]: Yes you are right. Now updated.

[the remark this answers is not in the document]
Comment (gberaudo) [reply geoportailv3/static/less/offline.less]: There was actually an issue with your code. Now fixed.

[the remark this answers is not in the document]
Comment (fgravin) [reply geoportailv3/static/less/offline.less]: what was wrong ?

[the remark this answers is not in the document]
Comment (gberaudo) [reply geoportailv3/static/less/offline.less]: It conflicted with the other comma separated rules that come above.

**Pull request merged**